loader/loader.c

```diff
@@ -6718,13 +6770,31 @@ VkResult setup_loader_term_phys_dev_groups(struct loader_instance *inst) {
         cur_icd_group_count += count_this_time;
     }
 
+#if defined(__linux__) || defined(__QNXNTO__) || defined(__FreeBSD__)
+    bool sort_items = true;
+    char *env_value = loader_getenv("VK_LOADER_DISABLE_SORT", inst);
```

Comment:
Is this the same code as above? This should be its own function, DRY and all that. Should be a simple thing that returns a bool on whether it should sort or not.